srcpkgs/Uranium/template

```diff
@@ -15,3 +15,4 @@ license="LGPL-3.0-or-later"
 homepage="https://github.com/Ultimaker/Uranium"
 distfiles="https://github.com/Ultimaker/Uranium/archive/${version}.tar.gz"
 checksum=36896e4807d1ce7040b7ea3cb1288a9e175d55e03e1485b19ba09b407e9515e8
+nocross="python3-scipy nocross"
```

Comment:
Does it make sense to add broken warnings for things that depend on other packages that are already broken? If we manage to cross compile Scipy, someone will have to remove all these nocross warnings too. From my quick testing, trying to cross compile Uranium fails as soon as it starts looking for the `python3-scipy` package.